### PR TITLE
Fix validation layers errors

### PR DESCRIPTION
**Pull request by Karina-Murawko-Wisniewska-Mobica**
## Description

Fix validation layers errors

### CODE REVIEW

File: samples/extensions/extended_dynamic_state2/extended_dynamic_state2.cpp [hunk 1]
```diff
@@ -534,8 +530,10 @@ void ExtendedDynamicState2::build_command_buffers()
  */
 void ExtendedDynamicState2::create_descriptor_pool()
 {
+	const uint32_t frame_number = static_cast<uint32_t>(draw_cmd_buffers.size());
```
Comment (Szymon-Rajczakowski) [inline samples/extensions/extended_dynamic_state2/extended_dynamic_state2.cpp]: I think it should be frame**s**_number.

File: samples/extensions/extended_dynamic_state2/extended_dynamic_state2.cpp [hunk 2]
```diff
@@ -534,8 +530,10 @@ void ExtendedDynamicState2::build_command_buffers()
...
+
 	std::vector<VkDescriptorPoolSize> pool_sizes = {
-	    vkb::initializers::descriptor_pool_size(VK_DESCRIPTOR_TYPE_UNIFORM_BUFFER, 3),
+	    vkb::initializers::descriptor_pool_size(VK_DESCRIPTOR_TYPE_UNIFORM_BUFFER, frame_number * 3),
```
Comment (Szymon-Rajczakowski) [inline samples/extensions/extended_dynamic_state2/extended_dynamic_state2.cpp]: Why does the number of frames only affect the uniform buffer?

**Review by Szymon-Rajczakowski — CHANGES_REQUESTED** [review]

File: samples/extensions/extended_dynamic_state2/extended_dynamic_state2.cpp (hunk 1 above)
Comment (Karina-Murawko-Wisniewska-Mobica) [reply samples/extensions/extended_dynamic_state2/extended_dynamic_state2.cpp]: fixed

**Review by Szymon-Rajczakowski — APPROVED** [review]

File: samples/extensions/extended_dynamic_state2/extended_dynamic_state2.cpp (hunk 2 above)
Comment (Karina-Murawko-Wisniewska-Mobica) [reply samples/extensions/extended_dynamic_state2/extended_dynamic_state2.cpp]: changed to reflect allocation

**Pull request merged**